src/scripts/ui_main.gd

```diff
@@ -7,6 +7,34 @@ func _ready() -> void:
 	_g.connect("ui_scale_decrease", self, "ui_scale_down")
 	SaveLoadSettings.connect("settings_loaded", self, "_on_settings_loaded", [], CONNECT_ONESHOT)
 	SaveLoadSettings.load_settings()
+	
+	# Check errors of previous session
```

Comment:
Maybe I missed it - but shouldn't the local storage variable "error" be cleared after sending this Error event?

Reply:
I moved it below making the post